src/auth.js

```diff
@@ -83,16 +83,18 @@ export function refresh (config) {
   })
 }
 
-export function auth () {
+export function auth (force) {
```

Comment:
```suggestion
export function auth (force = false) {
```
Safer than assuming auth() will be called with a parameter every time, since it actually works without. Unless... #2 :eyes: